frontend/model/contracts/identity.js

```diff
@@ -62,7 +62,9 @@ sbp('chelonia/defineContract', {
       process ({ data }, { state }) {
         const initialState = merge({
           settings: {},
-          attributes: {},
+          attributes: {
+            allowDMInvite: true
+          },
```

Comment:
Since users can create DM and make someone else join the DM, I think all users have risk that others can control their workload.
We don't have any design for this, so this should be removed at the moment?
IMHO, it is necessary but if you want to remove it, I will.

Reply:
This is something that needs to be worked through in a separate issue/PR together with @corrideat, so while that’s a great observation, yes please remove it for now and open an issue so that we can tackle it later using cryptography